docs/project/2024/kaan.ugur/index.md

```diff
@@ -0,0 +1,166 @@
+  **4-LCD Display:** The LCD display serves as the user interface, showcasing the current temperature in Celcius. Connected to the Rasberry Pi Pico via I2C.
+  
+  **5-Buzzer:** The buzzer serves as an audible alert system. Connected to the Rasberry Pi Pico via PWM. The buzzer's fequency and intensity can be modulated to emit different warning sounds based on the temperature level.
+  
+  **6-LED Indicator:** The LED indicator offers a visual cue to accompany the audible alert. It is directly controlled by the Rasberry Pi Pico's GPIO pins.
+  
+  **7-Button:** When pressed, the button triggers the Rasberry Pi Pico to activate the cooling fan. Connected to the Rasberry Pi Pico's GPIO pins.
+  
+
+## Log
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+*Hardware Components used in the Smart Summer Hat project:*
+ 
+  **1-Rasberry Pi Pico:** Microcontroller unit responsible for managing all operations.
+  
+  **2-Temperature Sensor:** Monitors ambient temperature and communicates with the Rasberry Pi Pico
+  
+  **3-DC Motor (Fan):** Provides airflow for cooling.
+  
+  **4-LCD Display:** Shows the current temperature in Celcius.
+  
+  **5-Buzzer:** Emits warning sounds.
+  
+  **6-LED:** Provides visual alerts.
+  
+  **7-Button:** Enable user interaction, triggering actions such as activating the cooling fan.
+  
+  **8-A Hat:** A prototype hat to make the project portable.
+
+### Schematics
+
+Place your KiCAD schematics here.
+
+### Bill of Materials
+
+<!-- Fill out this table with all the hardware components that you might need.
+
+The format is 
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico H](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [42 RON](https://www.optimusdigital.ro/ro/placi-raspberry-pi/12393-raspberry-pi-pico-h.html?search_query=pico+H&results=32) |
+| [BMP 280](https://cdn-shop.adafruit.com/datasheets/BST-BMP280-DS001-11.pdf) | Temperature Sensor | [10 RON](https://www.optimusdigital.ro/ro/senzori-senzori-de-presiune/1777-modul-senzor-de-presiune-barometric-bmp280.html?search_query=bmp280&results=11) |
+| [DC Motor (FAN)](https://datasheetspdf.com/datasheet-pdf/917194/775-5520F-CC.html) | DC Motor for FAN | [10 RON](https://www.bitmi.ro/motor-dc-3v-6v-pentru-proiecte-electronice-10651.html?gad_source=1&gclid=EAIaIQobChMIhsGZpandhQMVEplQBh0msAG5EAQYCCABEgLS5vD_BwE) |
+| [1602 Display](https://html.alldatasheet.com/html-pdf/1574132/CRYSTAIFONTZ/LCD-1602A/1009/1/LCD-1602A.html) | LCD Display | [25 RON](https://www.bitmi.ro/electronica/ecran-lcd1602-cu-modul-i2c-iic-10487.html) |
+| [Jumper Wires (Female - Female)]() | Female - Female Wires for connections | [7 RON](https://www.bitmi.ro/componente-electronice/40-fire-dupont-mama-mama-30cm-10503.html) |
```

Comment:
```suggestion
| Jumper Wires (Female - Female) | Female - Female Wires for connections | [7 RON](https://www.bitmi.ro/componente-electronice/40-fire-dupont-mama-mama-30cm-10503.html) |
```
 if there is no link

Reply:
I have changed the lines how you suggested and have updated.